bindings/py/tests/encoders/scalar_encoder_test.py

```diff
@@ -23,8 +23,8 @@
 import pytest
 import time
 
-from htm.bindings.encoders import ScalarEncoder, ScalarEncoderParameters
-from htm.bindings.sdr import SDR, Metrics
+from htm.encoders.scalar_encoder import ScalarEncoder, ScalarEncoderParameters
+from htm import SDR, Metrics
```

Comment:
nit, why this version compared to the "pure" bindings? I assume some plot,debug functionality?

Reply:
I don't want the users to need to import the bindings.  Instead I want everything to appear as a single python package, with the bindings as a hidden/unused backend.  I think it would be confusing for the users to see imports from both `htm.encoders` and `htm.bindings.encoders` because then they won't know where to look to find things.